test/Android/androidtest.cmd

```diff
@@ -36,9 +37,12 @@ if NOT exist "%_GlassDir%glass2.exe" echo Getting Glass from NuGet.& call "%_Pro
 if NOT "%ERRORLEVEL%"=="0" echo ERROR: Failed to get Glass from NuGet.& exit /b -1
 
 :: Copy binary to folder "Microsoft.VisualStudio.Glass" which required by glass2.exe in runtime for Visual Studio 2017
-if defined VS150COMNTOOLS (
+if defined VS140COMNTOOLS (
```

Comment:
>VS140COMNTOOLS [](start = 11, length = 14)

I don't understand this change

Reply:
It was 150 and that isnt' specified in VS2019. we check for the VS140 one above in the file so i thought htere was some other reason for the check. I'll change it to VSINSTALLDIR